docs/get-started/how-it-works.md

```diff
@@ -1,12 +1,9 @@
-# How Does it Work?
-
-## Contrastive metric learning
+# {octicon}`question` How Does it Work?
 
 From an algorithmic perspective,
-**Finetuner** leverages a contrastive metric learning approach to improve your model.
-How does it work?
+Finetuner leverages the contrastive approach to improve your model for similarity matching.
 
-### Step 1: Convert a model into an embedding model
+## Step 1: Build embedding model
 
 Finetuner interprets the backbone model architecture,
```

Comment:
I think for many users it is not clear what the term "backbone model architecture" means